torch/distributed/fsdp/flat_param.py

```diff
@@ -2383,9 +2383,12 @@ def _check_storage_freed(tensor: Tensor):
         )
 
     @staticmethod
-    def _check_storage_allocated(tensor: Tensor):
+    def _check_storage_allocated(tensor: Tensor, fqns: List[str]):
```

Comment:
We should fix the type either via `Tuple[str, ...]` or `Iterable[str]` or similar.
```suggestion
    def _check_storage_allocated(tensor: Tensor, fqns: Tuple[str, ...]):
```